File: dsa-practice/arrays/codes/problemset1/prob1.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    int n=5;
    int arr[n] = {2,4,5,6,3};
    int mx = arr[0];
    for (int i = 0; i < n; i++)
    {
        mx = max(mx,arr[i]);
        cout<<mx<<endl;
        
    }
    cout<<"Maximum:- "<<mx<<endl;
    return 0;
}

// Time complexity== O(n);

File: dsa-practice/arrays/codes/problemset1/prob1.js
var mx = function (arr) {
    var m = arr[0];
    arr.forEach(function (e, i) {
        if (m < e) {
            m = e;
        }
    });
    return m;
};
console.log(mx([1, 2, 56, 23, 124, 135, 67]));
